app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    files_names = command.split()
+    first_file, second_file = files_names[1], files_names[2]
```

Comment:
check if files_names has 3 elements **before** it, because code will be failed if files_names has less then 2 arguments

Reply:
Got it!